components/Recipe/Overview/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import { Recipe } from 'lib/types'
+import Image from 'next/image'
+import styles from 'components/Recipe/Overview/overview.module.css'
+
+import starLabel from 'public/images/Extra/star_label.png'
+import hintPlate from 'public/images/Extra/hint-plate.png'
+
+import Buttons from 'components/Recipe/Buttons'
+import EquipmentList from 'components/Recipe/EquipmentList'
+import IngredientsList from 'components/Recipe/IngredientsList'
+import CategoryInfo from 'components/Recipe/CategoryInfo'
+
+interface ParamTypes {
+    recipe: Recipe
+}
+
+/**
+ * A page displaying an overview of a particular recipe as specified in the URL.
+ * It includes information such as the recipe's category, tags, ingredients and
+ * equipment. From here, a user can navigate to pages displaying the recipe's
+ * steps in a slideshow or one page format.
+ */
+const RecipeOverview: React.FC = ({ recipe }) => {
+    // console.log(recipe.hint)
+    // console.log(styles['recipe-name'])
```

Comment:
Good practice to remove these before committing to avoid cluttering code for others